build.gradle.kts

```diff
@@ -1,17 +1,26 @@
-import org.gradle.internal.impldep.org.junit.experimental.categories.Categories.CategoryFilter.exclude
-
 plugins {
     id("uk.gov.justice.hmpps.gradle-spring-boot") version "4.9.0"
-    kotlin("plugin.spring") version "1.8.0"
+    kotlin("plugin.spring") version "1.9.0"
```

Comment:
Might be an idea to go straight to version 2 and upgrade the `uk.gov.justice.hmpps.gradle-spring-boot` then you won't need to add lines 8-15

Reply:
https://github.com/ministryofjustice/hmpps-person-record/blob/cc31fd159bf29480a795edac149ea9c4197d58cf/build.gradle.kts#L12